YoutubeDLSharp/Converters/DateTimeConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace YoutubeDLSharp.Converters
+{    
+    public class UnixTimestampConverter : JsonConverter<DateTime?>
```

Comment:
To make sure everything works as intended for this converter, it would be great to test one of the properties where it is used in the metadata tests.